src/main/java/org/springframework/data/hazelcast/repository/query/HazelcastPropertyComparator.java

```diff
@@ -33,6 +36,29 @@
         implements Comparator<Entry<?, ?>>, Serializable {
     private static final long serialVersionUID = 1L;
 
+    private static final MethodHandle EXTRACT_VALUE_HAZELCAST_41 = resolveExtractValueHazelcast41();
+    private static final MethodHandle EXTRACT_VALUE_HAZELCAST_403 = resolveExtractValueHazelcast403();
+
+
+    private static MethodHandle resolveExtractValueHazelcast41() {
+        try {
+            return MethodHandles.lookup().findStatic(ReflectionHelper.class,
+              "extractValue", MethodType.methodType(Object.class, Object.class, String.class, boolean.class));
+        } catch (Throwable ex) {
+            return null;
+        }
+    }
+
+    private static MethodHandle resolveExtractValueHazelcast403() {
+        try {
+            return MethodHandles.lookup()
+              .findStatic(ReflectionHelper.class,
+                "extractValue", MethodType.methodType(Object.class, Object.class, String.class));
```

Comment:
```suggestion
            return MethodHandles.lookup().findStatic(ReflectionHelper.class,
                "extractValue", MethodType.methodType(Object.class, Object.class, String.class));
```

For the consistent formatting.